tools/tests/test_inline_pack.conf

```
@@ -26,7 +26,9 @@
       "platform": "all",
       "version": "1.5.0",
       "discovery": [
-        "select pid from processes where name = 'foobar';"
+        "select pid from processes where name = 'foobar';",
+        1,
+        2
```

```suggestion
        2,
        [],
        {"a": 1}
```

Some more not-strings